oracle12/src/test/java/org/jdbi/v3/oracle12/TestOracleReturning.java

```diff
@@ -44,7 +45,7 @@ public class TestOracleReturning {
     static final String CONTAINER_VERSION = "gvenzl/oracle-free:" + System.getProperty("oracle.container.version", "slim-faststart");
 
     @Container
-    static OracleContainer oc = new OracleContainer(CONTAINER_VERSION);
```

Comment:
Since we now customize this in a bunch of places, could move it to a static factory method